src/components/datepicker/js/calendar.js

```diff
@@ -57,7 +57,8 @@
       scope: {
         minDate: '=mdMinDate',
         maxDate: '=mdMaxDate',
-        dateFilter: '=mdDateFilter'
+        dateFilter: '=mdDateFilter',
+        _currentView: '@mdCurrentView'
```

Comment:
Can you not make this optional and do in the controller

```js
this.currentView = this.currentView || 'month';
```

This would drop the unnecessary variable